recipes/classix/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: classix
+  version: 0.1.1
```

Comment:
This version appears not to match the version you're specifying below in the source.

Reply:
Hi Joshua, is it necessary to be consistent with PyPI?

Reply:
Okay, I will pull a new request to fit this with ``ClassixClustering``